usr/src/cmd/truss/fcall.c

```diff
@@ -1513,12 +1503,10 @@ function_return(private_t *pri, struct callstack *Stk)
 	uintptr_t fp = Lsp->pr_reg[R_FP];
 	int i;
 
-#ifdef _LP64
```

Comment:
Is there a reason some of this other stuff isn't wrapped up in MULTI_DATAMODEL like in the prior truss files? That's true for most of this file.

Reply:
Only that it wasn't necessary to do.  I can do it if you'd like

Reply:
OK, so the thing that made it necessary or not was depending on whether or not it basically would still compile. Is that correct?

Reply:
Compile and work!  But I can make this match up if you like

Reply:
No, I don't need to create more work. I just wanted to understand and make sure I was capturing the conversions correctly.